src/Fixer/ControlStructure/TrailingCommaInMultilineFixer.php

```diff
@@ -54,6 +54,11 @@ final class TrailingCommaInMultilineFixer extends AbstractFixer implements Confi
      */
     public const ELEMENTS_PARAMETERS = 'parameters';
 
+    /**
+     * @internal
+     */
+    public const ELEMENTS_MATCH = 'match';
```

Comment:
I'm thinking if ELEMENTS_MATCH is enough self describing term. Match by itself has no commas at the end of the line, but it's expressions has them. Maybe ELEMENT_MATCH_EXPRESSIONS would be more verbose?

Reply:
Fair point, thanks, I've updated the PR :+1: